packages/boot/package.json

```diff
@@ -35,7 +35,7 @@
     "@endo/far": "^0.2.19",
     "@endo/promise-kit": "^0.2.57",
     "@endo/stream": "^0.3.26",
-    "import-meta-resolve": "^2.2.1"
+    "import-meta-resolve": "^3.0.0"
```

Comment:
I didn't parse out the argument against upgrading it.

Reply:
The downside is that the shim will perform synchronous I/O (to read `package.json` files). The native implementation doesn’t need to read or interpret `package.json` files because we can be sure Node.js already has them memoized. It had to read them in order to link the surrounding module.

The downside of inducing I/O in these cases is tenuous. It will reduce event loop availability, which is a good reason to avoid them while serving traffic or other operations. If tail latency for a server is not an issue, synchronous I/O tends overall to be faster than concurrent I/O.

I don’t fully grasp an absolute censure for I/O for OCaps. If we’re generally using this mechanism on boot-time or test-time constants, there’s not much risk of an attacker gaining the ability to indirectly tease these programs into reading more `package.json` files than they strictly need. But this mechanism could induce an exception.

I’d say: be careful with dynamic import, even dynamic resolution. Because it requires care, as a reviewer, I would rather see `import.meta.resolve` where it’s getting used rather than `pkgAbsPath`. It raises a flag I need to see.